buildspec.yml

```diff
@@ -1,4 +1,7 @@
 version: 0.2
+env:
+  secrets-manager: 
+    SAFETY_KEY: "/codebuild/safety/key"
```

Comment:
can we avoid setting env variables in buildspec and instead do directly on CB job?

Reply:
using multi part docker build instead